src/data-seeding/certificates/handler.ts

```diff
@@ -13,36 +13,8 @@ import { Request, ResponseToolkit } from '@hapi/hapi'
 import { readFileSync } from 'fs'
 
 export async function certificateHandler(request: Request, h: ResponseToolkit) {
```

Comment:
Can you move this handler outside data-seeding directory please?

Reply:
Yeah. Did it in country-config PR. Also closed this one.